Makefile

```diff
@@ -0,0 +1,10 @@
+STATIC_SRC_DIR=wagtail_localize/static_src
+
+npm_deps:
+	cd ${STATIC_SRC_DIR} && npm install
+
+build: npm_deps
+	cd ${STATIC_SRC_DIR} && npm run build
+
+dev: npm_deps
```

Comment:
This will run `npm install` every time we want to watch files, won't it?

Reply:
Yep, it finishes really quickly when it hasn't got anything to do so I kinda like running it every time I start the watcher